test/e2e/tests/custom-rpc-history.spec.js

```diff
@@ -160,7 +160,15 @@ describe('Stores custom RPC history', function () {
         });
 
         await rpcUrlInput.clear();
-        await rpcUrlInput.sendKeys(newRpcUrl);
+
+        // We cannot use sendKeys() here, because a network request will be fired after each
+        // keypress, and the privacy snapshot will show:
+        // `New hosts found: l,lo,loc,loca,local,localh,localho,localhos`
+        // In the longer term, we may want to debounce this
+        await driver.pasteIntoField(
+          '.form-field:nth-of-type(2) input[type="text"]',
+          newRpcUrl,
+        );
```

Comment:
Curious... why was this not caught before I wonder. Regardless good catch

Reply:
Hah, yeah, not totally sure, but probably proxy-related.  And let's debounce this later!